src/buildifier/buildifier.ts

```diff
@@ -131,7 +131,7 @@ export function getBuildifierFileType(fsPath: string): BuildifierFileType {
   if (parsedPath.ext === ".oss") {
     parsedPath = path.parse(parsedPath.name);
   }
-  if (parsedPath.ext === ".bzl" || parsedPath.ext === ".sky") {
+  if (parsedPath.ext === ".bzl") {
     return "bzl";
   }
```

Comment:
Looks like a more faithful reimplementation of the buildifier code would be:

```suggestion
  switch (parsedPath.ext) {
    case ".bzl":
      return "bzl";
    case ".sky":
      return "default";
  }
```